src/components/repository-profile.component.js

```diff
@@ -164,13 +164,17 @@ export const RepositoryProfile = ({
       </Text>
 
       {repository.fork && (
-        <Text style={[styles.subtitle, styles.subtitleFork]}>
+        <Text
+          nativeId="repository-fork-container"
```

Comment:
I had to add this prop so it was possible to target this specific component in its test file.
`nativeId` is used for e2e tests [Docs About nativeId](https://facebook.github.io/react-native/docs/text.html#nativeid)